pkg/scannerv4/mappers/mappers.go

```diff
@@ -357,11 +365,54 @@ func sortBySeverity(ids []string, vulnerabilities map[string]*v4.VulnerabilityRe
 	})
 }
 
-func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircore.Vulnerability, nvdVulns map[string]map[string]*nvdschema.CVEAPIJSON20CVEItem) (map[string]*v4.VulnerabilityReport_Vulnerability, error) {
+// rhelVulnsEPSS gets highest EPSS score for each Red Hat advisory name
+// TODO(ROX-27729): get the highest EPSS score for an RHSA across all CVEs associated with that RHSA
+func rhelVulnsEPSS(vulns map[string]*claircore.Vulnerability, epssItems map[string]epss.EPSSItem) map[string]epss.EPSSItem {
```

Comment:
I believe I will be redoing this in the CSAF enricher PR since it'll have the list of CVEs related to each RHSA, so I'm going to mostly ignore this function